general-superstaq/general_superstaq/check/configs.py

```diff
@@ -12,16 +12,18 @@
 @check_utils.enable_exit_on_failure
 def run(
     *args: str,
-    config_file: str = "setup.cfg",
+    config_file: str = "pyproject.toml",
     ignore_match: str = "# REPO-SPECIFIC CONFIG",
+    start_match: str = "# Package-specific configuration (e.g., build specification) ends here.",
```

Comment:
imo it'd be cleaner/make the pyproject files more readable if we just used a line like:
```suggestion
    start_match: str = "# Check configuration:",
```